task1/task.py

```diff
@@ -1,3 +1,6 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    for x in range(1,10):
```

Comment:
you should include the `10` itself now its till `10`

Reply:
Nice comment @tomkidoz , @ntohidi do this

Reply:
Done